tests/behat/Paragraphs/WYSIWYG.feature

```diff
@@ -0,0 +1,153 @@
+    And I should not see an "option" element in the "content" region
+    And I should not see a "textarea" element in the "content" region
+    And I should not see a "fieldset" element in the "content" region
+    And I should not see a "legend" element in the "content" region
+    And I should not see a "address" element in the "content" region
+
+    # Headers
+    And I should not see an "h1:contains('Level 1 heading')" element
+    And I should see an "h2:contains('Level 02 Heading')" element
+    And I should see an "h3:contains('Level 03 Heading')" element
+    And I should see an "h4:contains('Level 04 Heading')" element
+    And I should see an "h5:contains('Level 05 Heading')" element
+    And I should see an "h6:contains('Level 06 Heading')" element
+
+    # Text Tags
+    And I should see an "p:contains('A small paragraph')" element
+    And I should see an "a:contains('Normal Link')" element
+    And I should see an "a.su-button:contains('Button')" element
+    And I should see an "a.su-button--big:contains('Big Button')" element
+    And I should see an "a.su-button--secondary:contains('Secondary Button')" element
+    And I should see an "em:contains('emphasis')" element
+    And I should see an "strong:contains('important')" element
+    And I should see 1 "blockquote" element in the "content" region
+    And I should not see a "footer" element in the "content" region
+    And I should see 2 "code" elements in the "content" region
+    And I should see 1 "dl" element in the "content" region
+    And I should see 2 "dt" element in the "content" region
+    And I should see 2 "dd" element in the "content" region
+
+    # List Tags
+    And I should see an "ul li:contains('This is a list')" element
+    And I should see an "ul ul li:contains('child list items')" element
+    And I should see an "ol li:contains('Ordered list item')" element
+    And I should see an "ol ol li:contains('Child ordered list item')" element
+
+    # Table Tags
+    And I should see 1 "table" element in the "content" region
+    And I should see 1 "caption" element in the "content" region
+    And I should see 1 "tbody" element in the "content" region
+    And I should see 3 "tr" elements in the "content" region
+    And I should see 2 "th[scope]" elements in the "content" region
+    And I should see 4 "td" element in the "content" region
+
+  @api @javascript
+  Scenario: Test Embedded Image
+    Given I am logged in as a user with the "administrator" role
+    And I am on "/node/add/stanford_page"
+    Then I fill in "Title" with "Test WYSIWYG Paragraph"
+    And I press "Add WYSIWYG"
+    And I wait for AJAX to finish
+    Then I click the "a[title='Embed Media']" element
+    And I wait for AJAX to finish
+    Then I switch to "entity_browser_iframe_media_browser" iframe
+    Then I click "Embed a File"
+    And I wait for AJAX to finish
+    Then I drop "../assets/images/logo.jpg" file into dropzone
```

Comment:
I am getting a Javascript error "Dropzone is undefined" on this step.

Reply:
Seems that the step `And I wait for AJAX to finish` isn't waiting long enough.

Reply:
I got that once. if we get it often, we could just put in some extra code into the step definition to wait for `Dropzone` to be available.